test/spec/spectest.cpp

```diff
@@ -14,18 +14,10 @@
 ///
 //===----------------------------------------------------------------------===//
 
-#define RAPIDJSON_HAS_STDSTRING 1
-#if defined(__SSE4_2__)
-#define RAPIDJSON_SSE42 1
-#elif defined(__SSE2__)
-#define RAPIDJSON_SSE2 1
-#elif defined(__ARM_NEON__) || defined(__ARM_NEON) || defined(__ARM_NEON_FP)
-#define RAPIDJSON_NEON 1
```

Comment:
Does simdjson provide such options?

Reply:
Simdjson actually automatically detects and sets these options while compiling, for the specific architecture, as per this [doc](https://github.com/simdjson/simdjson/blob/master/doc/implementation-selection.md)